runbot_gitlab/templates/build.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <template id="build_button_inherit" inherit_id="runbot.build_button">
+        <!-- Change title: Open commit on GitHub -> GitLab -->
+        <xpath expr="//a[@aria-label='Open commit on GitHub']" position="attributes">
```

Comment:
For being more resilient, I would do:

- Add on current buttons an invisible condition to be hidden when they are not GitHub.
- Add new buttons with the new labels for Gitlab
- If you have a new backend like Bitbucket, it should add the same

Reply:
@pedrobaeza  
I like the idea, it sounds more scalable. However, that would need to be a separate work, because it would imply a little of refactoring

That button I'm inheriting, along with other ones, are already inherited and modified by this same module, e.g.:

https://github.com/OCA/runbot-addons/blob/62a839f8ab8f3af2994790f4fa27d149317fd493/runbot_gitlab/templates/runbot_gitlab_logos.xml#L14-L17

Actually, that's why I took this approach, to be more consistent with what it's already done.

Reply:
Then switch the other template to the same technique.

Reply:
@pedrobaeza & @moylop260 
I applied the proposed approach. Could you review, please?

Sorry for the delay, I've been so busy these days. But I'm in the game again.